easypost/requestor.py

```diff
@@ -91,7 +120,11 @@ def request_raw(
                 f"at {SUPPORT_EMAIL} for assistance."
             )
 
-        abs_url = "%s%s" % (api_base, url or "")
+        if beta:
```

Comment:
Let's maybe flesh out a better approach to supporting the Beta base URL in the future. This will probably come up when we do versioning.